src/lib.rs

```diff
@@ -105,6 +111,22 @@ impl<'a> TreeItem<'a> {
         }
     }
 
+    pub fn child(&self, index: usize) -> Option<&Self> {
+        if index < self.children.len() {
+            Some(&self.children[index])
+        } else {
+            None
+        }
```

Comment:
I think we can use the method of `self.children` directly which does exactly the logic by itself and don't reinvent the wheel. Or am I missing something here?
```suggestion
        self.children.get(index)
```

Reply:
This was a small oversight on my part. Using `.get()` indeed makes the intent clearer.